packages/jest-core/package.json

```diff
@@ -8,6 +8,7 @@
     "@jest/console": "^24.3.0",
     "@jest/reporters": "^24.5.0",
     "@jest/test-result": "^24.5.0",
+    "@jest/test-sequencer": "^24.5.0",
```

Comment:
why does `jest-core` depend on this? It should be injected from `jest-config`, I think

Reply:
I import it in `runJest.ts` for type checking.

Reply:
please add it as dev dependency instead

Reply:
When add it as dev dependency, it will cause `import/no-extraneous-dependencies` error.
Is it  a good way to fix it by adding inline `disable-eslint` ?

Reply:
Yeah I think we have that in a few other places already